Files/Helpers/JumpListManager.cs

```diff
@@ -59,50 +60,70 @@ private void AddFolder(string path)
         {
             if (instance != null && !JumpListItemPaths.Contains(path))
             {
-                string displayName;
-                if (path.Equals(CommonPaths.DesktopPath, StringComparison.OrdinalIgnoreCase))
+                string displayName = null;
+                if (path.EndsWith("\\"))
                 {
-                    displayName = "ms-resource:///Resources/SidebarDesktop";
+                    // Jumplist item argument can't end with a slash so append a character that can't exist in a directory name to support listing drives.
+                    var drive = App.DrivesManager.Drives.Where(drive => drive.Path == path).FirstOrDefault();
+		    if (drive == null)
```

Comment:
Could you check the formatting?
![image](https://user-images.githubusercontent.com/9673091/144473217-afd861a6-e429-4806-ad02-2cf369638158.png)

Reply:
This is the problem when using the GitHub web editor. I'll fix it.